### ISSUE TITLE
Publicize: Custom Messages that are copy-pasted into the Custom Message metabox are ignored.

### ISSUE THREAD

**Issue by jeremypaavola** [Automattic/wp-calypso#29813]
When copy-pasting a custom message into Publicize, the custom message is ignored and instead WP's auto-generated excerpt is pushed to Facebook. If copy-pasting and then afterwards editing the message (such as removing and re-adding a letter), then custom excerpt is pushed as expected.

Reported in 1666206-zen

#### Steps to reproduce
1. Go to WP Admin
2. Navigate to Settings --> Sharing (ensure Jetpack is set up, and Publicize active)
3. Connect a Facebook account
4. Go to Posts --> Add New
5. Enter a title and text into the editor
6. Ensure pre-check is Enabled in the Block Editor (https://jetpack.com/support/publicize/#publicize-in-the-block-editor), then click 'Publish' and copy-paste a custom message into the custom message box
8. Publish the post
9. View the post on Facebook

#### What I expected

I expect the custom message to show

#### What happened instead

Facebook published with WP's auto-excerpt instead. (if, in step 6, the copy-pasted message is interacted with, Facebook will post the expected message. Something happens during the edit that causes the Block Editor to see the copy-pasted message.)

**Comment by arunsathiya** [comment]
@jeremypaavola I tried to reproduce this, but I am unsuccessful. I tested with Twitter, The copy/content I paste in the Publicize field appears on the tweet, as expected.

[Here is a video showing the same](https://cloudup.com/ceUbpYHU_gp). Should I test specifically with Facebook?

As a side note, I think this belongs better on `Automattic/jetpack` as this involves Jetpack's Publicize feature and is concerning /wp-admin's Block Editor. [Calypso editor](https://wordpress.com/post)/Calypsoified Gutenberg/[Gutenlypso](https://wordpress.com/block-editor/post) does not seem to be related in any way, but I choose to defer to developers for a clarity on this.
